src/libpgmoneta/wal.c

```diff
@@ -508,6 +522,16 @@ pgmoneta_wal(int srv, char** argv)
    exit(1);
 }
 
+char*
```

Comment:
missing `static`

Reply:
Thanks for letting me know. I fixed it in the latest push.